bundles/org.openhab.binding.modbus.kermi/src/main/resources/OH-INF/thing/xcenter-state-channel-types.xml

```diff
@@ -25,5 +25,9 @@
 		<item-type>Contact</item-type>
 		<label>Global Alarm State</label>
 		<description>If there's a global alarm</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>OpenState</tag>
```

Comment:
Remove the property as it seems  a generic alarm (maybe the item type created this suggestion)